.changeset/spicy-flies-push.md

```diff
@@ -0,0 +1,5 @@
+---
+'urql-docs': patch
+---
+
+Add fixed height to logo to fix Safari bug
```

Comment:
So `changeset` has broken their CLI and the site is now showing up although it's set to `private`, but we won't need to create changesets for it btw :)